Recommend Hogpatch or Hedge House for offsites and onboarding (#16984)
PostHog/posthog.com: contents/handbook/company/offsites.md

Reviewer: One thing to consider with this change is that it turns work travel from a perk/benefit to more of a downside. People often get tired of going to the same places all the time. 

I do see the benefits of defaulting to the two places where other posthoggers are. 

Did something inspire this update? Have you noticed wasted travel recently?

Reviewer 2: +1; IMO this turns something that I felt was a perk and a cool feature of working at PostHog into more of a chore. 

I’m coming from a company that similarly limited Phoenix and SF as the 2 offsite locations and can confirm everyone was so over it and after a while people just ended up looking for excuses not to go (after my 6th time in Phoenix you truly can't pay me to go back there nowadays)

This makes a lot of sense for onboardings and I assumed this was already the case, but for small team offsites this feels like a downgrade/it's a shame.

I think with this change it would be worth clarifying what's the expectation in terms of what people should be getting out of their small team offsites; 

I thought the whole point was more dedicated work time with your team and adjacent teams, not broad in-person collaboration. The value I got from the content/marketing/website one last year was precisely the fact we had isolated, focused, alone time to deep dive into the themes that matter to us and we barely had time to cover that over the week + hang out with each other — I don’t think trying to weave in extra face time with other people and other teams would make these necessarily more valuable or more productive in terms of moving forward our specific agendas, but maybe that's not the point/I misunderstood what these are meant to accomplish

Reviewer 2: [In fairness I have a personal bias against this decision because for visa and general state of the world reasons I have no interest in going to the States anytime soon, and the thought of having to keep going back to SF for my offsites (which is the location that logistically would make sense for my team out of these 2 options) makes me dread them more than look forward to them lol]

Author: This came partly from our all-company offsite, partly from other conversations I've had. In person high bandwidth hanging out is just _so good_. I love being a remote company, but I think we're missing an opportunity by having onboardings or small teams go to random places with no other posthog people. The more we can get people in the same room, the more crazy ambitious ideas we'll come up with.

On offsites being 'a perk'... I get that attitude but it's also a little bit short sighted. Yes offsites should be fun, but ultimately they are a business trip, not a holiday. SF and London are cool places, and we've invested _a lot_ in having spaces there that are nice and you want to spend time in. I think hanging out with other PostHog people makes it worth it?

Author: two more thoughts:
- SF in particular is just somewhere I believe everyone at PostHog should spend more time. It's the centre of one of the biggest revolutions in tech in our lifetime. hogpatch literally is in the same neighorhood as openai, and hogpatch is full of YC founders and users. The more time people spend here, the more they soak up the atmosphere, the better we'll do
- the offsite you talked about is probably one of the bigger non-all company offsite, so I get your feeling of there being a lot on. most small team offsites aren't like that and would benefit more from hanging out with others, though I believe even in a big offsite like that having other perspectives would be super valuable

Reviewer 2: >  In person high bandwidth hanging out is just so good. 

Oh trust me I agree, I'm all for spending more in person time with coworkers whenever possible, so much so that I'm headed to Toronto next week to work with PostHog people, and headed to London in June to work from HedgeHouse for 2 weeks, all of this on my own dime, just because of how much I believe in and enjoy/crave being in the same room with others in the company regardless of team, and _do_ think it's worth it.

So I'm personally a huge believer in spending intentional time ad-hoc with other coworkers (and act accordingly), that's not what I'm arguing against.

My criticism is specifically on forcing the small team offsites to happen in either SF or London every single time, regardless on whether that's convenient or cost-effective for team members, which feels very weirdly rigid & I'm struggling to fully grasp the upsides; maybe if it was like "switch around, 1 year do in one of the offices, other year pick a location", but this feels like a super narrow directive for reasons I'm still struggling to comprehend

Then again I think this policy change would benefit from being appended with the new vision/goal behind small team offsites

Curious what that would mean for example for a team that is already based in one of the origin cities (i.e. London), are their offsite options to either stay in London or to fly 12 hours to SF? 

> Yes offsites should be fun, but ultimately they are a business trip, not a holiday.

FWIW I don't see it as a holiday either since there's zero to no free time, but before it was a business trip with something different/exciting to look forward to (for example, getting to see Lisbon through the eyes of PostHog locals was really special to me), now it's just... a corporate business trip. Which is fine, these _are_ still cool cities and seeing people will always be nice, but it is undeniably shift towards the corporate norm and a lot less special/weird/fun in the way I've gotten to know PostHog

Reviewer 3: I fundamentally agree with the directive here - I think it makes sense for us all to gather where most of the PostHog team is. I am probably a bit old school - I think this is work travel/this is a work place, so have no qualms with being tied down to specific locations.

BUT, I think there are real world conditions that would play a big role that may be overlooked here. I know @timgl you already added an exception for visa issues. But I also massively +1 @natalia-amorim 's point here

> [In fairness I have a personal bias against this decision because for visa and general state of the world reasons I have no interest in going to the States anytime soon, and the thought of having to keep going back to SF for my offsites (which is the location that logistically would make sense for my team out of these 2 options) makes me dread them more than look forward to them lol]

I know at PostHog we don't take a stand - and that's fine. But the visa exception doesn't cover the broader 'don't want to travel to the US right now' situation a lot of people are in - I am in that boat, I just don't feel safe travelling to the US right now.

But my team's composition is going to directly collide with this directive (as I'm sure will be the case for many). Everyone in Finance except me, is now going to be based in the US - which makes me _dread_ the travel setup. Feels like I'll have to somehow just be ok with not feeling safe and travel to the US for work (which **has not** been my experience at/doesn't sound like PostHog _at all_ in my experience), or have everyone in North America fly down to London (more $$). As opposed to maybe compromising and meeting in Toronto, for instance.

When folks met up in Feb in Toronto, a whole bunch of folks showed up there - Engineering, Ops, Marketing - from Toronto/GTA/Montreal/Ottawa/the US as well. The WeWork conference room was _full_. +1 to @brandonleung 's point [here](https://posthog.slack.com/archives/C06LMMS3YP4/p1779418271070199?thread_ts=1779399367.316289&cid=C06LMMS3YP4) - I think as we hire more across geographies, we have specific hubs being created with a decent chunk of folks. I think this directive can have flexibility built into it to let teams make the best decision for themselves.

We want to try to encourage small teams to get together once each year. These are more focused on work and on creating strong bonds within teams. Ideally they are spaced appropriately through the year in relation to the all-company offsite.

> With very few exceptions, team offsites should happen at [Hogpatch](/handbook/people/hogpatch-operations) in San Francisco or the [Hedge House](#hedge-house) in London. Both are designed for it, remove most of the planning friction, and you'll always overlap with other PostHog teams — which is way more fun, and means you'll meet more people.

Author: ```suggestion
With very few exceptions, team offsites should happen at [Hogpatch](/handbook/people/hogpatch-operations) in San Francisco or the [Hedge House](#hedge-house) in London. Both are designed for it, remove most of the planning friction, and you'll always overlap with other PostHog teams — which is way more fun, and means you'll meet more people.

If that's not practical because of visas or excessive travel time, feel free to pick another location that makes more sense.
```

Author: @rafaeelaudibert how about this?

> With very few exceptions, team offsites should happen at [Hogpatch](/handbook/people/hogpatch-operations) in San Francisco or the [Hedge House](#hedge-house) in London. Both are designed for it, remove most of the planning friction, and you'll always overlap with other PostHog teams — which is way more fun, and means you'll meet more people.

Reviewer 4: I like the idea of this being the default place for work travel, saves the back and forth of figuring out where we should go and allows folks to hang out with each other more often. Also, it makes sense that it's the place for onboarding.

For team offsites this is a bit annoying, as it's nice to make those something special to spend time with your team, and it being in London / SF where we'll have been 3 times already that year won't feel like that anymore.

It being expressed as a kind of mandatory with very few exceptions thing feels a bit corporate and definitely makes work travel feel like it's not a benefit as Raquel said 🤷‍♂️

Reviewer 5: I do agree this feels more control than context driven initiative. 

However, the net benefit from doing offsites in these places are much better, reliable working environment, cross-pollination, usually better travel links. 

I do still think there is space for wider teams to do offsites in unique locations:- 

- The GTM offsite
- The Marketing
- AI teams
- Data Stack 

Where there is a large enough group and actually those teams needing their own space and needs etc. 

However where it is a handful of people, defaulting to our hubs makes a million times more sense. 

This would then make it kind of look like in a normal year of working at PostHog:- 

- 1 company offsite in unique location
- 1 wider team offsite in unique location 
- 1 small team offsite in SF or London or Cambridge
- other ad-hoc meetups in SF or London or Cambridge 

Also the title of this PR is "Recommend" but the wording is not a recommendation, it's a directive.

> With very few exceptions, team offsites should happen at [Hogpatch](/handbook/people/hogpatch-operations) in San Francisco or the [Hedge House](#hedge-house) in London. Both are designed for it, remove most of the planning friction, and you'll always overlap with other PostHog teams — which is way more fun, and means you'll meet more people.

Reviewer 6: ```suggestion
We want team offsites to be social events and encourage you to optimize offsites for meeting as many colleagues as is reasonably possible, even if it means spending more money or travelling a bit further. [Hogpatch](/handbook/people/hogpatch-operations) in San Francisco or the [Hedge House](#hedge-house) in London are both great options for this and we strongly encourage them as a default choice. 
```

I have read this thread many times and been asked about it many times in DM and I think this is representative of what we're really trying to achieve.

Author: > asked about it many times in DM 
make it public!

agree on the wording. just adding some of the reasons why we're doing this etc:

```suggestion
We want team offsites to be social events and encourage you to optimize offsites for meeting as many colleagues as is reasonably possible, even if it means spending more money or travelling further. [Hogpatch](/handbook/people/hogpatch-operations) in San Francisco or the [Hedge House](#hedge-house) in London should be the default choice, because:

- They're places where there are already a lot of PostHog people that you can hang out with. The more we have random cross-polination, the more ambitious ideas we'll come up with
- If most teams do their offsites there, you're much more likely to run into even more people
- We've put a lot of money and effort into making Hogpatch and Hedge House beautiful spaces that you want to work from
- SF has the benefit of being the epicenter of everything happening in tech, and we have lots of YC founders working out of Hogpatch who you can meet and learn from

You can still grab an AirBnB in any of these places to have more team time if you prefer, over a hotel.
```

> With very few exceptions, team offsites should happen at [Hogpatch](/handbook/people/hogpatch-operations) in San Francisco or the [Hedge House](#hedge-house) in London. Both are designed for it, remove most of the planning friction, and you'll always overlap with other PostHog teams — which is way better for cross-team collaboration than going somewhere new.

Author: ```suggestion
With very few exceptions, team offsites should happen at [Hogpatch](/handbook/people/hogpatch-operations) in San Francisco or the [Hedge House](#hedge-house) in London. Both are designed for it, remove most of the planning friction, and you'll always overlap with other PostHog teams — which is way more fun, and means you'll meet more people.
```

## Hedge House

> PostHog runs two Hedge Houses in the UK - a small one in Cambridge and a larger one in London. They are actual houses (yes, with a few bedrooms attached!) designed for small teams to run their offsites, host in-person onboardings, or come together for larger internal events like hackathons. Anyone at PostHog is welcome to use them as much as they like — alongside [Hogpatch](/handbook/people/hogpatch-operations) in San Francisco, these are our recommended spots for team offsites and in-person onboarding.

Author: ```suggestion
PostHog runs two Hedge Houses in the UK - a small one in Cambridge and a larger one in London. They are actual houses (yes, with a few bedrooms attached!) designed for small teams to run their offsites, host in-person onboardings, or come together for larger internal events like hackathons. Anyone at PostHog is welcome to use them as much as they like — alongside [Hogpatch](/handbook/people/hogpatch-operations) in San Francisco, these are where you should do team offsites and in-person onboarding.
```
